### PR TITLE
#2559. Fix metadata tests

### PR DESCRIPTION
**Pull request by sgrekhov**

### CODE REVIEW

File: LanguageFeatures/Augmentation-libraries/metadata_A01_t02.dart [hunk 1]
```diff
@@ -68,13 +71,14 @@ extension type ET(int id) {
...
-  var symbol = MirrorSystem .getSymbol("topLevelVariable");
+  var symbol = MirrorSystem.getSymbol("topLevelVariable");
   DeclarationMirror varMirror =
-    libraryMirror.declarations[symbol] as DeclarationMirror;
-  Expect.equals('.Meta1',
-      MirrorSystem.getName(varMirror.metadata[0].type.qualifiedName));
-  Expect.equals('.Meta2',
-      MirrorSystem.getName(varMirror.metadata[1].type.qualifiedName));
+      libraryMirror.declarations[symbol] as DeclarationMirror;
+  print(MirrorSystem.getName(varMirror.metadata[0].type.qualifiedName));
```
Comment (eernstg) [inline LanguageFeatures/Augmentation-libraries/metadata_A01_t02.dart:77]: Leftover debugging code?

**Review by eernstg — COMMENTED** [review]
Looks very good, thanks! I think one line might be a remnant from debugging?

This means that we won't complain if the order of metadata objects differs from what we'd expect by looking at the syntax, and that's fine: There is no language specification rule to prevent that. (Also, it seems harmless in practice.)

File: LanguageFeatures/Augmentation-libraries/metadata_A01_t02.dart (hunk 1 above)
Comment (sgrekhov) [reply LanguageFeatures/Augmentation-libraries/metadata_A01_t02.dart]: Yes, thank you! Removed.

**Review by eernstg — APPROVED** [review]
LGTM

**Pull request merged**